src/theory/bv/bitblast/simple_bitblaster.cpp

```diff
@@ -129,6 +129,17 @@ Node BBSimple::getModelFromSatSolver(TNode a, bool fullModel)
   return utils::mkConst(bits.size(), value);
 }
 
+void BBSimple::computeRelevantTerms(std::set<Node>& termSet)
+{
+  for (const auto& var : d_variables)
```

Comment:
Maybe assert that we are in eager bit-blast mode and add a comment on why it is only necessary in that case?